test/unit/spec/common/validate-Spec.js

```diff
@@ -178,11 +178,6 @@ define(['bluebird', 'common/validation', 'widgets/appWidgets2/validators/constan
 
         // STRING
 
-        function wrap(val) {
```

Comment:
This was used to wrap otherwise synchronous validators for no apparent reason. Removed and unwrapped those that used it.

Reply:
😖